### PR TITLE
Convoy's Game of life

### PR DESCRIPTION
**Pull request by zeal2end**
A simple mini project made using python and curses based on Convoy's game of life.

# Description

A short summary of what is included in your Pull Request.

Fixes #(issue_no)

- [ ] Issue_no: ```<insert your issue no here>```
- [x] Not applicable 

## Type of change

Choosing one or more options from the following as per the nature of your Pull request.

- [x] New project insert
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Documentation Update

## Project

- #### **Your Project Name: Convoy's Game OF Life** 

- #### **Short Description:** A simple mini project made for Command line and based on the theme of Convoy's Game of life.(A zero player game)

# Checklist:
Please tick all the boxes that are fulfilled by your Pull Request.

- [x] I have named my files and folder, according to this project's guidelines.
- [x] My code follows the style guidelines of this project.
- [x] I have commented on my code, particularly in hard-to-understand areas.
- [x] I have created a helpful and easy to understand `README.md`, according to the given [`README_TEMPLATE.`](https://github.com/nduongthucanh/python-mini-project/blob/master/README_TEMPLATE.md)
- [x] My changes do not produce any warnings.

### CODE REVIEW

**Review by sidba2003 — CHANGES_REQUESTED** [review]
I believe more commenting should be added to the GameOfLife.py for completely new people to start working with. Overall seems like a nice project which I will definitely like to learn more about provided the code wasn't hard to follow.

**Review by ndleah — APPROVED** [review]
